app.py

```diff
@@ -398,6 +465,137 @@ def index():
         'index.html'
     )
 
+
+
+
+
+# Subscribing/unsubscribing user from club
+
+@app.route('/subscriptions', methods=['GET'])
+def subscriptions():    
+    Session, engine = dbconnect(db_options)
+    session = Session()
+
+    subscriptions = session.query(UserToClubMapping).all()
+    
+    ret_subs = []
+    for s in subscriptions:
+        ret_subs.append(
+            {
+                'id': s.id,
+                'user_id': s.user_id,
+                'club_id': s.club_id,
+                'created_on': s.created_on,
+            }
+        )
+        #users = map(lambda u: dict(u), users)
+    return jsonify(ret_subs)
+
+
+@app.route('/subscribe', methods=['POST'])
+def subscribe():
+    if request.mimetype != 'application/json':
+        raise Exception('Content-Type is not "application/json".')
+    j = request.get_json()
+    Session, engine = dbconnect(db_options)
+    session = Session()
+    link = UserToClubMapping(
```

Comment:
sync with ricky, he deleted this table in favor of UserClubPostitionMapping
I think we decided to use the one wiht position, so do a find and replace for that here